stubs/requests/requests/sessions.pyi

```diff
@@ -1,14 +1,12 @@
 from _typeshed import Self, SupportsItems
-from typing import IO, Any, Callable, Iterable, Mapping, MutableMapping, TypeVar, Union
+from typing import IO, Any, Callable, Iterable, Mapping, MutableMapping, Union
 from typing_extensions import TypeAlias
 
-from urllib3 import _collections
+from urllib3._collections import RecentlyUsedContainer
```

Comment:
```suggestion
from urllib3._collections import RecentlyUsedContainer as RecentlyUsedContainer
```

Reply:
I did that initially, but stubtest called me out. Turns out that it doesn't actually exist at runtime, we just use it for annotations in the stub. Not sure why stubtest didn't flag it before now.

Source: https://github.com/psf/requests/blob/main/requests/sessions.py

Stubtest: https://github.com/python/typeshed/runs/6046468647?check_suite_focus=true

Reply:
Oh interesting. stubtest's checking of type aliases is a little loose, see https://github.com/python/mypy/blob/ce858a6a0c85206ff877faf702293a9b67fae750/mypy/stubtest.py#L960 (I don't quite remember why I did it this way, maybe worth reinvestigating)